kernel/gic/src/gic/mod.rs

```diff
@@ -132,6 +148,13 @@ impl GicRegisters {
 
 const_assert_eq!(core::mem::size_of::<GicRegisters>(), 0x1000);
 
+fn current_redist_index() -> usize {
```

Comment:
nit: give this a name (and ideally, some doc comments too) that indicate it is the redist index for the current _cpu_, not just any other type of "current" concept